sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncConflict.java

```diff
@@ -106,4 +106,33 @@ String getLink() {
         builder.append(getId());
         return builder.toString();
     }
+
+    private Mono<CosmosConflictResponse> readInternal(RequestOptions options, Context context) {
```

Comment:
Can the internal methods be combined here?

Reply:
Done

Reply:
Done